src/locales/en.json

```diff
@@ -35,7 +35,7 @@
     "preset name": "name",
     "step password empty": "Password's value is missing.",
     "step password too weak": "Your password is too weak. You should add an uppercase, a number, a special character or a longer password.",
-    "step password title": "Step 3: Choose your password",
```

Comment:
Yup, it's related to https://trello.com/c/oFbhV3ET/410-1-cc-7-amélioration-suite-aux-interviews-utilisateurs-locale, I took it to let @CPatchane go an other tasks :wink: